src/routes/common/logging/__tests__/test.logging.module.ts

```diff
@@ -0,0 +1,26 @@
+import { Global, Module } from '@nestjs/common';
+import { ILoggingService, LoggingService } from '../logging.interface';
+
+const loggerService: ILoggingService = {
+  log: function (message: any, ...optionalParams: any[]) {
+    console.log(message, ...optionalParams);
+  },
+  error: function (message: any, ...optionalParams: any[]) {
+    console.error(message, ...optionalParams);
+  },
+  warn: function (message: any, ...optionalParams: any[]) {
+    console.warn(message, ...optionalParams);
+  },
+  debug: function (message: string, ...optionalParams: any[]): void {
+    console.debug(message, ...optionalParams);
+  },
```

Comment:
We need not specify the types again here:

```suggestion
  log: console.log,
  error: console.error,
  warn: console.warn,
  debug: console.debug,
```